nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/computation/AbstractBatchComputation.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * (C) Copyright 2018 Nuxeo SA (http://nuxeo.com/) and others.
```

Comment:
This is just `Nuxeo` and not `Nuxeo SA` in the copyright if I recall

Reply:
then I made the mistake everywhere, thanks I will fix that in another commit